dotnet/src/SemanticKernel.Abstractions/Orchestration/FunctionResult.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Orchestration;
+
+/// <summary>
+/// Function result after execution.
+/// </summary>
+public sealed class FunctionResult
+{
+    /// <summary>
+    /// Additional data from model.
```

Comment:
Can you expand on this description to explain when this would be populated. The original comment was a bit vague also but a description of how multiple model results get populated would be useful e.g., if there are nested semantic functions which one is first and last in the list.

Reply:
Thanks, updated. There is not a lot of information to put there since this field is populated immediately after function execution. The reason why it's a collection is because one AI response can have multiple results. I described it in the documentation.